python/tests/aggregate_tests/test_binary_max.py

```diff
@@ -0,0 +1,67 @@
+from .aggtst_base import TstView
+
+
+class aggtst_binary_max_value(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [{"c1": "312b541d", "c2": "63141f4d"}]
+        self.sql = """CREATE MATERIALIZED VIEW binary_max AS SELECT
+                      MAX(c1) AS c1, MAX(c2) AS c2
+                      FROM binary_tbl"""
+
+
+class aggtst_binary_max_gby(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [
+            {"id": 0, "c1": "17382115", "c2": "37424d58"},
+            {"id": 1, "c1": "312b541d", "c2": "63141f4d"},
+        ]
+        self.sql = """CREATE MATERIALIZED VIEW binary_max_gby AS SELECT
+                      id, MAX(c1) AS c1, MAX(c2) AS c2
+                      FROM binary_tbl
+                      GROUP BY id"""
+
+
+class aggtst_binary_max_distinct(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [{"c1": "312b541d", "c2": "63141f4d"}]
+        self.sql = """CREATE MATERIALIZED VIEW binary_max_distinct AS SELECT
+                      MAX(DISTINCT c1) AS c1, MAX(DISTINCT c2) AS c2
+                      FROM binary_tbl"""
+
+
+class aggtst_binary_max_distinct_gby(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [
+            {"id": 0, "c1": "17382115", "c2": "37424d58"},
+            {"id": 1, "c1": "312b541d", "c2": "63141f4d"},
+        ]
+        self.sql = """CREATE MATERIALIZED VIEW binary_max_distinct_gby AS SELECT
+                      id, MAX(DISTINCT c1) AS c1, MAX(DISTINCT c2) AS c2
+                      FROM binary_tbl
+                      GROUP BY id"""
+
+
+class aggtst_binary_max_where(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [{"c1": "17382115", "c2": "63141f4d"}]
+        self.sql = """CREATE MATERIALIZED VIEW binary_max_where AS SELECT
+                      MAX(c1) FILTER(WHERE c1 < c2) AS c1, MAX(c2) FILTER(WHERE c1 < c2) AS c2
+                      FROM binary_tbl"""
+
+
+class aggtst_binary_max_where_gby(TstView):
```

Comment:
why do you need to check manually?
doesn't this work in Postgres?

Reply:
I couldn't apply MAX directly on BYTEA in Postgres. This was a possible workaround: 
`SELECT  MAX(ENCODE(col, 'hex')) FROM tbl; `
Since the dataset was small, I manually verified the result. Would you have preferred I used this workaround for comparison instead?

Reply:
interesting that Postgres does not support max for binary.
This is fine.